osiris/authentication/BUILD

```diff
@@ -69,7 +69,7 @@ go_library(
     ],
 )
 go_test(
-    name = "go_default_test",
+    name = "server-test",
     srcs = ["server_test.go"],
     embed = [":go_default_library"],
```

Comment:
Why did we rename everything to default... and here he other way around? Just curious :)

Reply:
Because it is a test, not a library and the name is printed during test runs.